locale/ua.po

```diff
@@ -0,0 +1,1446 @@
+# #-#-#-#-#  openscad-tmp.pot (OpenSCAD 2018.08.15)  #-#-#-#-#
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the OpenSCAD package.
+# Kyrylo Romanenko <romanenko-kv@hotmail.com>, 2018.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"#-#-#-#-#  openscad-tmp.pot (OpenSCAD 2018.08.15)  #-#-#-#-#\n"
+"Project-Id-Version: OpenSCAD 2018.08.15\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2018-08-15 15:07+0200\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: Kyrylo Romanenko <romanenko-kv@hotmail.com>\n"
+"Language-Team: Ukrainian\n"
+"Language: ua\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=INTEGER; plural=EXPRESSION;\n"
```

Comment:
The error report suggests
`"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2);\n"
`
which roughly matches Russian plural forms
https://github.com/openscad/openscad/blob/e1f043d2d038a2c1839ec39354383b5de2b12cdb/locale/ru.po#L16-L17

Reply:
Thank you for good example! In Ukrainian it should go like this, depending from number:
1 - 4: msgstr[0]
5 - 20: msgstr[1]
21-24: msgstr[0]
25-30: msgstr[1]
31-34: msgstr[0]
35-40: msgstr[1]
...
I think this should work as expected:
"Plural-Forms: nplurals=2; plural=(n%10>=1 && n%10<=4 && (n%100<11 || n%100>14) ? 0"
" : n%10==0 && n%10>=5 && n%10<=9 ? 1);\n"

I am not a programmer, so it's a bit tricky for me :)